backend/api/v1/acl.go

```diff
@@ -84,6 +85,42 @@ func (c *aclStreamingConn) Receive(msg any) error {
 	return c.StreamingHandlerConn.Receive(msg)
 }
 
+// hasAllowMissingEnabled checks if the request has AllowMissing field set to true.
+// Uses reflection to handle different request types generically.
+func hasAllowMissingEnabled(request any) bool {
+	if request == nil {
+		return false
+	}
+
+	v := reflect.ValueOf(request)
+
```

Comment:
Since request is a protobuf message, can we use protoreflect instead?

something like
```
		pm := request.Any().(proto.Message)
		pm.ProtoReflect().Descriptor().Fields().ByName("allow_missing")
```